main.tf

```diff
@@ -162,12 +169,16 @@ resource "google_sql_database_instance" "gitlab_db" {
     ip_configuration {
       ipv4_enabled    = "false"
       private_network = google_compute_network.gitlab.self_link
+      require_ssl     = "true"
     }
 
     backup_configuration {
-      enabled                        = true
-      start_time                     = "02:00"
+      enabled                        = var.postgresql_enable_backup
+      start_time                     = var.postgresql_backup_start_time
       point_in_time_recovery_enabled = true
+        backup_retention_settings {
+          retained_backups           = var. postgresql_backup_retained_count
+        }
```

Comment:
Maybe it is only a render problem, but I see this code block not correctly aligned

Reply:
Changed the indenting, but tflint does not show me any error.  :(